multichain-testing/scripts/update-config.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+## see https://github.com/cosmology-tech/starship/blob/1d60f55c631b4d0f92a43ad92e9a935298aa3aa5/starship/charts/devnet/scripts/default/update-config.sh
+
+CHAIN_ID="${CHAIN_ID:=osmosis}"
+CHAIN_DIR="${CHAIN_DIR:=$HOME/.osmosisd}"
+KEYS_CONFIG="${KEYS_CONFIG:=configs/keys.json}"
+
+set -eux
+
+ls $CHAIN_DIR
+
+
+echo "Update config.toml file"
+sed -i -e 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:26657"#g' $CHAIN_DIR/config/config.toml
+sed -i -e 's/index_all_keys = false/index_all_keys = true/g' $CHAIN_DIR/config/config.toml
+sed -i -e 's/seeds = ".*"/seeds = ""/g' $CHAIN_DIR/config/config.toml
+sed -i -e 's#cors_allowed_origins = \[\]#cors_allowed_origins = \["*"\]#g' $CHAIN_DIR/config/config.toml
+
+echo "Increase `*_bytes` parameters for MsgInstallBundle"
+sed -i -e "s/max_body_bytes = .*/max_body_bytes = 15728640/g" $CHAIN_DIR/config/config.toml
+sed -i -e "s/max_header_bytes = .*/max_header_bytes = 1048576/g" $CHAIN_DIR/config/config.toml
+sed -i -e "s/max_txs_bytes = .*/max_txs_bytes = 1073741824/g" $CHAIN_DIR/config/config.toml
+sed -i -e "s/max_tx_bytes = .*/max_tx_bytes = 15728640/g" $CHAIN_DIR/config/config.toml
+sed -i -e "s/^rpc-max-body-bytes =.*/rpc-max-body-bytes = 15728640/" $CHAIN_DIR/config/app.toml
```

Comment:
15728640  is 15 mebibytes. For whatever reason, [agoric-cli's chain-config.js](https://github.com/Agoric/agoric-sdk/blob/master/packages/agoric-cli/src/chain-config.js#L66) has 15 megabytes (15e6).

This value shouldn't be any higher. Please reference the agoric-cli value, either by importing or the comment.